test/ssl.jl

```diff
@@ -104,6 +104,12 @@ function test_cmd(ctxt::ConnContext)
 end
 
 @testset "ssl" begin
-    ssl_tests(true)
-    ssl_tests(false)
+    ssl_tests(true, true)
+    ssl_tests(true, false)
+
+    ENV["FTPS"] = true
```

Comment:
I'd probably use a `withenv` block here to avoid contaminating the environment outside of these specific tests.

Reply:
Will do